Source/GSXib5KeyedUnarchiver.m

```diff
@@ -2789,6 +2790,32 @@ - (id) decodeObjectForXib: (GSXibElement*)element
         [object setHeaderToolTip: [element attributeForKey: @"toolTip"]];
     }
 
+  // Handle state for NSSwitch
```

Comment:
As the code for NSSwitch has gotten a lot bigger now it would be better to move it into a separate method and add a special decoder for the tag "switch".
But it could also just be that it would be sufficient to add "enabled" to the array "XmlBoolDefaultYes". I really don't understand the special handling here.

Reply:
It would be better to put it in a common place.  I was unaware of XmlBoolDefaultYes until now.

Reply:
Done